pkg/broker/broker.go

```diff
@@ -101,16 +101,15 @@ func (a AnsibleBroker) getServiceInstance(instanceUUID uuid.UUID) (*apb.ServiceI
 
 }
 func (a AnsibleBroker) Login() error {
-	a.log.Debug("Retrieving serviceaccount token")
-	token, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/token")
+	clientConfig, err := k8srestclient.InClusterConfig()
```

Comment:
I don't think that will work.
I'm assuming a.clusterConfig is an instance of apb.ClusterConfig.

apb.ClusterConfig doesn't contain the variables we need.
https://github.com/openshift/ansible-service-broker/blob/master/pkg/apb/client.go#L45


I was looking for a way to reference this:
https://github.com/openshift/ansible-service-broker/blob/master/pkg/apb/client.go#L82